IM_calculation/scripts/calculate_ims.py

```diff
@@ -163,6 +163,21 @@ def load_args():
     )
 
     args = parser.parse_args()
+
+    if constants.Components.ceas.str_value in args.components:
+        wrong_ims = ""
+        if "FAS" not in args.im:
+            wrong_ims = ",".join(args.im)
+        else:
+            # there are IMs other than FAS but user only wants EAS
+            # without this check, the code still runs, but the non-FAS IMs
+            # won't be shown in the output csv without a notice/warning
+            if len(args.im) > 1 and len(args.components) == 1:
+                args.im.remove("FAS")
+                wrong_ims = ",".join(args.im)
+        if wrong_ims:
+            parser.error("EAS is not available with the specified IM types: {}".format(wrong_ims))
```

Comment:
I think it might be useful to Include how to rectify. "adding FAS as a specified IM will allow EAS to be calculated." or...
But it might make the error message quite long.

Reply:
I have already revised the error message, which implies how to resolve. See below.

Reply:
Perfect. That encomapasses both cases.

That'll teach me for reading commits individually.

Reply:
Sorry about the confusion, I changed my mind after making one commit.